nixos/modules/hardware/tuxedo-drivers.nix

```diff
@@ -3,13 +3,20 @@
 with lib;
 
 let
-  cfg = config.hardware.tuxedo-keyboard;
-  tuxedo-keyboard = config.boot.kernelPackages.tuxedo-keyboard;
+  cfg = config.hardware.tuxedo-drivers;
+  tuxedo-drivers = config.boot.kernelPackages.tuxedo-drivers;
 in
   {
-    options.hardware.tuxedo-keyboard = {
+    imports = [
+      (mkRenamedOptionModule [ "hardware" "tuxedo-keyboard" "enable" ] [ "hardware" "tuxedo-drivers" "enable" ])
```

Comment:
issue: this breaks the tuxedo-rs module, which enables tuxedo-keyboard. It will have to be updated.